crates/astria-bridge-withdrawer/src/bridge_withdrawer/mod.rs

```diff
@@ -71,13 +76,20 @@ impl BridgeWithdrawer {
     /// # Errors
     ///
     /// - If the provided `api_addr` string cannot be parsed as a socket address.
-    pub fn new(cfg: Config, metrics: &'static Metrics) -> eyre::Result<(Self, ShutdownHandle)> {
+    pub async fn new(
```

Comment:
I have a strong preference for this remaining sync (not async).

`BridgeWithdrawer` already has a setup phase during which async steps are run.